spec/controllers/container_service_controller_spec.rb

```diff
@@ -15,7 +15,11 @@
   it "renders show screen" do
     EvmSpecHelper.create_guid_miq_server_zone
     ems = FactoryGirl.create(:ems_kubernetes)
-    container_service = ContainerService.create(:ext_management_system => ems, :name => "Test Service")
+    container_service = ContainerService.create(
+      :ext_management_system => ems,
+      :container_project     => ContainerProject.create(:ext_management_system => ems, :name => "test"),
```

Comment:
adding a test project for the tested service. this was introduced due to the query in event_where_clause, in which a project 

> what does this do?

this adds a test project for the tested service. it was introduced due to query in event_where_clause, in which a project name is needed.